### ISSUE TITLE
Proposal: Add a validation set

### ISSUE THREAD

**Issue by lpf123321** [lpf123321/MLP-classifier#1]
Implement the K-fold validation and visualize the change of loss, training accuracy and validation accuracy in a graph.
